src/pointfree/toPromise.spec.js

```diff
@@ -0,0 +1,39 @@
+const test = require('tape')
+const sinon = require('sinon')
+const helpers = require('../test/helpers')
+
+const bindFunc = helpers.bindFunc
+
+const isFunction  = require('../core/isFunction')
+
+const constant = x => () => x
+
+const toPromise = require('./toPromise')
+
+test('toPromise pointfree', t => {
+  t.plan(13)
```

Comment:
We do not need `t.plan` anymore because it's no longer async.

Reply:
ahh ofcourse.... this is what i get for coding quickly and not checking :)